build.sbt

```diff
@@ -302,7 +302,7 @@ lazy val benchmarks = project
     kafkaScale := 1,
     buildInfoPackage := "akka.kafka.benchmarks",
     buildInfoKeys := Seq[BuildInfoKey](kafkaScale),
-    dockerComposeTestLogging := true,
```

Comment:
Is that intended to go in?

Reply:
Yes. When this is true, it prints all of the logs from docker containers (so Kafka starting up and shutting down) to the console. This makes it hard to locate the bench results.